src/sbvr-api/sbvr-utils.ts

```diff
@@ -1302,6 +1305,9 @@ const runRequest = (
 			}
 			throw err;
 		})
+		.tapCatch(err => {
+			return runHooks('ERROR', request.hooks, { req, request, tx, error: err });
```

Comment:
I think it could be worth putting this hook point after the `POSTRUN` hooks in case they throw an error it will allow handling, although there are potential arguments for both